CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+### Added
+- PRE and POST interaction commands
```

Comment:
it doesn't though, it's merely a blank command anyone can put what they want. In the config i've added shairport as an example command. 
It's disable by default because it requires setup of shairport and alsa. It's like this because you didn't want to be specific about shairport